### DISCUSSION TITLE
机器人偶尔会在不合适的时候加新反馈lable

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#1540] [提问] [answered]
比如 #1536 
以及很早以前有一个，issue号我忘了

**Comment by WForst-Breeze** [comment, marked as answer]
其实大部分只要操作程序得当就可以避免（）出现问题主要是在机器人执行操作的时间间隙中，如果 Collaborator 在这个机器人的反应间隙中更改了 label，机器人仍然会按照设定去添加它想添加的 label（我表达能力不行，可能说得有点迷，#1536 举个例子，我在 reopen 这个 issue 的时候，按照机器人的设定是要添加 `新反馈` label 并移除 `忽略` label 的，不过我在它执行这一步之前就先改了 label，但它仍然会依照设定去添加 `新反馈` label，所以才会出现这种情况。其他的情况也都大致相似）~不过如果能把它改的聪明点那就更好了~

**Reply by LTCatt** [reply]
这是因为 GitHub API 是使用梯子访问的，所以它在处理的时候有好几秒的延迟，这几秒里如果正好改了 Label 就会导致这种情况，没啥办法
